app/messaging/process-return-message.js

```diff
@@ -1,5 +1,10 @@
+const util = require('util')
+const { processReturnSettlement } = require('../inbound')
 const processReturnMessage = async (message, receiver) => {
   try {
+    const settlement = message.body
+    console.log('Return settlement processed:', util.inspect(settlement, false, null, true))
```

Comment:
Sommit tht I've hadda change of heart on is this log message

it makes it sound that the return settlement object/message/request has been processed, i.e. completed successfully -- however, we know that if `processReturnSettlement` throws it doesn't complete and in-fact fails and logs "Unable to process return message" which makes a lot more sense

therefore, my change of heart is that this log message should read, "Processing return settlement"